ingestion/functions/common/common_lib.py

```diff
@@ -36,6 +41,7 @@
 MIN_SOURCE_ID_LENGTH, MAX_SOURCE_ID_LENGTH = 24, 24
 
 logger = logging.getLogger(__name__)
+logger.setLevel("INFO")
```

Comment:
I think that rather than overriding the log level it might be better to set it in `ingestion_logging` based on environment--prod and qa get `INFO`, lower environments get `DEBUG`.

Reply:
Found a bug in setting loglevels in ingestion_logging (they are not followed, and loglevel is always rogging.WARNING), will create an issue for that, and include this task in that issue.